climpred/tests/test_stats.py

```diff
@@ -96,9 +97,8 @@ def test_dpp(PM_da_control_3d, chunk):
 )
 def test_potential_predictability_likely(PM_da_control_3d, func):
     """Check for positive diagnostic potential predictability in NA SST."""
-    control = PM_da_control_3d
-    print(control.dims)
-    res = func(control)
+    print(PM_da_control_3d.dims)
```

Comment:
Is this needed? Does this print every time or just print if it fails?

Reply:
in pytest only when it fails, but not needed